src/signer/noscrypt_signer.cpp

```diff
@@ -28,7 +45,7 @@ NoscryptSigner::NoscryptSigner(
 {
     plog::init(plog::debug, appender.get());
 
-    this->_reseedRandomNumberGenerator();
+    NostrSecureRng::reseed();
```

Comment:
Is reseeding an intended side-effect of calling the constructor? I believe this would affect the entire process that relies on openssl's default rng.

Reply:
I envisioned `NoscryptSigner` being a long-lived process, like a singleton, so the RNG would be reseeded once when Aedile starts up.  However, depending on how we actually use the signer class in practice, it may be better to move the reseed operation to Aedile's initializers, to ensure it's one-and-done for the duration of the process.

Reply:
Agreed. Removed from constructor as an unintended side-effect. Reseed is a public static function now, so we can just document it, or handle it in initializers. I think it depends on the use case. Some users may rely on OpenSSL in other parts of their projects and calling process-altering functions may not be intended/understood.